Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/States/InteractableStateModel.cs

```diff
@@ -8,18 +8,6 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// <summary>
     /// State data model, state management and comparison instructions
     /// </summary>
-
-    /*
-     * Have an enum with all the button states -
```

Comment:
This feels like information that might be useful to have *somewhere*.

Reply:
@killerantz Is there a rewording here that would be a little more helpful as a doccomment? The text here reads somewhat as dev notes to self, and as a reader I don't really know what to make of this.